gallery/src/components/NxDrawer/NxDrawerDisabledFunctionalityExample.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { NxDrawer,
+  NxButton,
+  NxP,
+  NxBackButton,
+  NxPageMain,
+  NxPageTitle,
+  NxH1,
+  useToggle
+} from '@sonatype/react-shared-components';
+
+export default function NxDrawerExample() {
+  const [showDrawerWithDisabledFunctionality, toggleDrawerWithDisabledFunctionality] = useToggle(false);
+  const [disabled, toggleDisabled] = useToggle(true);
+
+  return (
+    <>
+      <header className="nx-global-header">
+        <NxBackButton href="#/pages/Drawer" targetPageTitle="Documentation" />
+      </header>
+      <NxDrawer id="nx-drawer-disabled-functionality"
+                open={showDrawerWithDisabledFunctionality}
+                onClose={toggleDrawerWithDisabledFunctionality}
+                aria-labelledby="simple-drawer-disabled-title"
+                closeBtnDisabled={disabled}
+                closeBtnTooltip={disabled ? "Please save or revert changes" : "Close"}
```

Comment:
Let's not re-encode the default value in the calling code
```suggestion
                closeBtnTooltip={disabled ? "Please save or revert changes" : undefined}
```

Reply:
> Looks like an eslint error here as well, single quotes around 'Please save...'

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/1169/commits/c96e5c79f4a3de24f96ff7c74e4ed5d0ed842fb9